modules/calibration/data/ch/gnss_params/gnss_conf.pb.txt

```diff
@@ -0,0 +1,80 @@
+data {
+    format: NOVATEL_BINARY
+    serial {
+        device: "/dev/ttyACM0"
+        baud_rate: 460800
+    }
+}
+
+rtk_from {
+    format: RTCM_V3
+    ntrip {
+        address: "112.34.112.154"
+        port: 8000
+        mount_point: "BDKJ"
+        user: "user"
+        password: "password"
+        timeout_s: 5
+    }
+    push_location: true
+}
+
+rtk_to {
+    format: NOVATEL_BINARY
+    serial {
+        device: "/dev/ttyACM0"
+        baud_rate: 115200
+    }
+}
+
+command {
+    format: NOVATEL_BINARY
+    serial {
+        device: "/dev/ttyACM0"
+        baud_rate: 115200
+    }
+}
+
+rtk_solution_type: RTK_RECEIVER_SOLUTION
+imu_type: ADIS16488
```

Comment:
maybe you used newton m2? you can use the `gnss_conf_newton.pb.txt` default imu_type is `CPT_XW5651`